cookieninja/vcs.py

```diff
@@ -86,11 +88,21 @@ def clone(
 
     repo_url = repo_url.rstrip('/')
     repo_name = os.path.split(repo_url)[1]
+
+    repo_args = {
+        'git': ['git', 'clone'],
+        'hg': ['hg', 'clone'],
+    }
+    clone_command = repo_args[repo_type]  # avoid warning if defined in if-elif
     if repo_type == 'git':
         repo_name = repo_name.split(':')[-1].rsplit('.git')[0]
         repo_dir = os.path.normpath(os.path.join(clone_to_dir, repo_name))
-    if repo_type == 'hg':
```

Comment:
Why delete this line?

Reply:
This check is already covered in the map access above, so it isn't necessary. I actually removed it because it was getting the coverage percentage down to 99.7%, as it's impossible to trigger the code to not land on either the if or elif lololol.